ordersettler/ordersettler.go

```diff
@@ -200,10 +207,61 @@ func (r *OrderSettler) settleOrders(ctx context.Context) error {
 
 	lmt.Logger(ctx).Info("initiating order settlements", zap.Stringers("batches", toSettle))
 
-	if err = r.SettleBatches(ctx, toSettle); err != nil {
+	hashes, err := r.SettleBatches(ctx, toSettle)
+	if err != nil {
 		return fmt.Errorf("initiating order settlements: %w", err)
 	}
 
+	lmt.Logger(ctx).Info("order settlements initiated on chain", zap.Any("hashes", hashes))
+
+	return nil
+}
+
+// relaySettlements submits tx hashes for settlements to be relayed from the
+// settlements initiation chain (the orders destination chain), to the payout
+// chain (the orders source chain).
+func (r *OrderSettler) relaySettlements(
+	ctx context.Context,
+	txHashes []string,
+	batches []types.SettlementBatch,
+	submitter hyperlane.RelaySubmitter,
```

Comment:
need to pass in the submitter (aka a db tx in this case, but technically doesn't have to be) in order for this to be run in the same db tx